.github/ISSUE_TEMPLATE.md

```diff
@@ -4,15 +4,28 @@ Please don't file GitHub issues to ask questions! Instead use:
 * Stack Overflow: http://stackoverflow.com/questions/tagged/react-virtualized
 * The react-virtualized Slack channel: https://react-virtualized.now.sh
 
-# Reporting a Bug?
+**Do you want to request a *feature* or report a *bug*?**
 
-Please include either a failing unit test or a simple repro. You can start by forking one of these:
-* Code Sandbox: https://codesandbox.io/s/03qpzq1p9p?module=%2FExample.js
-* Plnkr: https://plnkr.co/edit/6syKo8cx3RfoO96hXFT1
-
-# Requesting a Feature?
+<!--
+Requesting a feature?
 
 Provide as much information as possible about your requested feature. Here are a few questions you may consider answering:
 * What's your use case? (Tell me about your application and what problem you're trying to solve.)
 * What interface do you have in mind? (What new properties or methods do you think might be helpful?)
 * Can you point to similar functionality with any existing libraries or components? (Working demos can be helpful.)
+
+Reporting a Bug?
+
+Please include either a failing unit test or a simple repro. You can start by forking one of these:
+* Code Sandbox: https://codesandbox.io/s/03qpzq1p9p?module=%2FExample.js
+* Plnkr: https://plnkr.co/edit/6syKo8cx3RfoO96hXFT1
+
+-->
+
+**What is the current behavior?**
+
+**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code and it doesn't have dependencies other than React. Paste the link to your JSFiddle (https://jsfiddle.net/Luktwrdm/) or CodeSandbox (https://codesandbox.io/s/new) example below:**
```

Comment:
These are the react code boxes, we should probably replace this with the ones we already had for this project?

https://codesandbox.io/s/03qpzq1p9p?module=%2FExample.js
https://plnkr.co/edit/6syKo8cx3RfoO96hXFT1